### PR TITLE
Added docker-compose logic

### PR DESCRIPTION
**Pull request by xRahul**
Related issue- https://github.com/chadfawcett/mailhound/issues/33
- [x] Added `docker-compose.yml` file
- [x] Added example environment file for docker
- [x] Updated readme to reflect the changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,19 +26,14 @@ Mailhound is a simple server side script for receiving form posts and emailing t
 3. Install mailhound with `npm install -g mailhound`
 4. Add config variable exports to the end of your `~/.profile` file
   1. `export SMTP=smtp://postmaster:pass@smtp.server.com`
-  2. `export KEY:name@domain.com`
+  2. `export ADMIN=name@domain.com`
```
Comment (chadfawcett) [inline README.md]: Thanks for changing this! I noticed it last night and thought I'd have to get to it.

File: README.md [hunk 2]
```diff
@@ -26,19 +26,14 @@ Mailhound is a simple server side script for receiving form posts and emailing t
...
-
-    ```
-    ADMIN=name@domain.com
-    SMTP=smtp://postmaster:pass@smtp.server.com
-    ```
-
-2. Run docker with `docker run --env-file=<path to the config file> -p 8000:8000 chadfawcett/mailhound`
+1. Create the config file `.env` from `.env.example` with your environment variables
+2. Run docker with `docker run --env-file=<path to the config file .env> -p 8000:8000 chadfawcett/mailhound`
+3. Alternatively run- `docker-compose up`
```
Comment (chadfawcett) [inline README.md]: Could this be broken up so it's not in the same list? Either add another ###subheading for docker-compose, or just have the list start over at 1.

> 1. ...
> 2. ...
>
> Alternatively, run with docker-compose
> 
> 1. `docker-compose up`

**Comment by chadfawcett** [comment]
Whoops. Also, could you please checkout these changes into a new branch and make a PR from that? I will have to update my contribution documentation to better outline the process.

**Comment by xRahul** [comment]
created new PR, so closing this one

**Pull request closed without merging**